shared_model/interfaces/base/signable.hpp

```diff
@@ -41,6 +42,10 @@ namespace shared_model {
 #define SIGNABLE(Model) Signable<Model, iroha::model::Model>
 #endif
 
+    /// Type of signature range, which returns when signatures are invoked
+    using SignatureRangeType = boost::any_range<const interface::Signature &,
+                                                boost::forward_traversal_tag>;
+
```

Comment:
I don't know if it should be moved to `interfaces/common_objects/types.hpp`?
In my opinion it fits here perfectly.